hbase-server/src/main/java/org/apache/hadoop/hbase/tool/Canary.java

```diff
@@ -843,9 +876,9 @@ public int run(String[] args) throws Exception {
               }
             }
             currentTimeLength = System.currentTimeMillis() - startTime;
-            if (currentTimeLength > this.timeout) {
+            if (currentTimeLength > timeout) {
```

Comment:
nit: avoid unrelated style changes like removing the `this` variables

Reply:
I removed bunch of class variables and store them in conf object.  I replaced the class variable with method local variable. So I can't use "this" anymore.